### ISSUE TITLE
This is a great thing that you publish can I encourage you toward creating a "loosening guide"

### ISSUE THREAD

**Issue by Leopere** [fastenhealth/fasten-onprem#315]
```
It can be as simple as

    Username: testuser
    Password: testuser
```
I think that when it comes to something that holds such important data for maintaining someone's privacy and bodily autonomy do you think that perhaps it would be prudent to publish the project in a way in which it's as secure as possible by default while still being runnable and then you publish a Loosening Guide as opposed to what people would normally publish called a Hardening Guide for security?

**Comment by AnalogJ** [comment]
Yeah this is a good point. This is definitely a hold-over from the BETA testing, we should get rid of that part completely.
